.github/workflows/release.yml

```diff
@@ -45,10 +44,21 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
-      - name: Install Helm
-        uses: azure/setup-helm@v1
+      # As of today, there is no way to filter/exclude charts to be released. Config does not support it:
+      # https://github.com/helm/chart-releaser/blob/4eb598f96aa53d1c0b8234af2087834e1f2275e1/pkg/config/config.go#L40-L61
+      # So we delete the Chart.yaml to filter the chart: https://github.com/helm/chart-releaser-action/pull/18
+      - name: remove the chart template as it should not be deployed
+        run: |
+          rm ./library/CHART-TEMPLATE/Chart.yaml
+
+      - name: Release library charts
+        uses: helm/chart-releaser-action@v1.2.1
```

Comment:
https://github.com/helm/chart-releaser-action/releases/tag/v1.3.0
maybe is worth to bump it.

Reply:
Solved in [d11b6ad](https://github.com/newrelic/helm-charts/pull/713/commits/d11b6ad0e194de9fe72561553c8986158bf11cf9)